Gordon360/Services/NewsService.cs

```diff
@@ -153,9 +184,161 @@ public StudentNewsViewModel EditPosting(int newsID, StudentNews newData)
             newsItem.Subject = newData.Subject;
             newsItem.Body = newData.Body;
 
+            if (newData.Image != null)
+            {
+                UploadImage(newData.Image, newsID);
+            }
+
+            //If the image property is null, it means the user removed whatever
+            //picture they had in their posting before.
+            else
```

Comment:
This is only true if the previous post had an image attached.

Reply:
Oh, you are correct- I'll clarify it in that comment.